examples/contrib/run_openai_gpt.py

```diff
@@ -182,9 +179,6 @@ def main():
     model.to(device)
 
     # Load and encode the datasets
-    if not args.train_dataset and not args.eval_dataset:
-        roc_stories = cached_path(ROCSTORIES_URL)
```

Comment:
I fail to see how the example uses this. I'm not seeing any code turning `roc_stories` into datasets.